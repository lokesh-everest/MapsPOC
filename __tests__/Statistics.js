import 'react-native';
import React from 'react';
import Statitics from "../app/Components/Statistics";

import renderer from 'react-test-renderer';

it('render correcly',()=>{
    const tree = renderer.create(<Statitics />);
    expect(tree).toMatchSnapshot();
})
