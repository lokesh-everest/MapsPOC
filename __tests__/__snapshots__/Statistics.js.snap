// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correcly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "white",
      }
    }
  >
    Distance:
     Km
  </Text>
  <Text
    style={
      Object {
        "color": "white",
      }
    }
  >
    Duration:
     min
  </Text>
</View>
`;
